packages/rainbowkit/src/utils/colors.ts

```diff
@@ -0,0 +1,30 @@
+export const convertHexToRGBA = (
+  hexCode: string,
```

Comment:
If we need colors to be provided in a hex format, we might need to add some validation when wallets are initially passed to RainbowKit, otherwise this code will silently break.

Reply:
i added some validation in `connectorsForWallets.ts`, will tag u there